source/guides/stacks/mers.rst

```diff
@@ -417,4 +506,53 @@ using pre-trained models and sample video files.
        ./gst-video-analytics/samples/shell/console_measure_fps_cpu.sh $VIDEO_EXAMPLES_DIR/bolt-detection.mp4
 
 
+Adding AOM support
+******************
+
+The current version of |MERS| does not include the `Alliance for Open Media
+<https://aomedia.org/>`_ Video Codec (AOM). AOM can be built from source on an
+individual basis.
+
+To add AOM support to the |MERS| image:
+
+
+#. The following tools are needed to add AOM support to |MERS|: **docker, git,
```

Comment:
"The following tools are needed to add AOM support to |MERS|: ** docker, git, patch**." --> "The ** docker, git, and 
 patch** tools are needed to add AOM support to |MERS|."